api/syfosmregister-v1.yaml

```diff
@@ -0,0 +1,355 @@
+openapi: 3.0.1
+
+info:
+  title: Syfosmregister API
+  description: API for sykmeldinger.
+  version: 1.0.0
+servers:
+  - url: 'https://api.example.com'
+paths:
+  /api/v1/sykmeldinger:
+    get:
+      operationId: getSykmeldingerForBruker
+      security:
+        - bearerAuth: []
+      tags:
+        - Sykmelding
+      parameters:
+        - $ref: '#/components/parameters/fnrQueryParam'
+      summary: Returnerer sykmeldinger for oppgitt fnr
+      responses:
+        '200':
+          $ref: '#/components/responses/SykmeldingListResponse'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+  /api/v1/sykmeldinger/{sykmeldingId}/bekreft:
+    post:
+      operationId: bekreftAvvistSykmelding
+      tags:
+        - Sykmelding
+      summary: Bekreft avviste sykmeldinger
+      parameters:
+        - $ref: '#/components/parameters/sykmeldingId'
+      responses:
+        200:
+          description: Ok
+
+  /sykmeldinger/{sykmeldingId}/status:
+    post:
+      operationId: addStatus
+      tags:
+        - Sykmelding status
+      summary: Api for å lagre status for sykmelding
+      parameters:
+        - $ref: '#/components/parameters/sykmeldingId'
+      responses:
+        201:
+          description: Created
+        401:
+          description: Unauthorized
+        403:
+          description: Forbidden
+      requestBody:
+        $ref: '#/components/requestBodies/StatusRequest'
+  /sykmeldinger/{sykmeldingId}/bekreft:
+    post:
+      operationId: bekreftSykmelding
+      tags:
+        - Sykmelding status
+      summary: API for bekrefting av sykmeldinger
+      parameters:
+        - $ref: '#/components/parameters/sykmeldingId'
+      responses:
+        201:
+          description: Created
+        401:
+          description: Unauthorized
+        403:
+          description: Forbidden
+      requestBody:
+        $ref: '#/components/requestBodies/SykmeldingBekreftEvent'
+  /sykmeldinger/{sykmeldingId}/send:
+    post:
+      operationId: sendSykmelding
+      tags:
+        - Sykmelding status
+      summary: API for sending av sykmeldinger
+      parameters:
+        - $ref: '#/components/parameters/sykmeldingId'
+      responses:
+        201:
+          description: Created
+        401:
+          description: Unauthorized
+        403:
+          description: Forbidden
+      requestBody:
+        $ref: '#/components/requestBodies/SykmeldingSendEvent'
+
+
+  /api/v1/internal/sykmeldinger:
+    get:
+      operationId: getSykmeldinger
+      tags:
+        - Sykmelding
+        - Internal
```

Comment:
Peker dette til path'en, eller hvem apiet er tilgjengelig for? For alle apiene for endring av status (bortsett fra bekreft for avviste sykmeldinger) er også kun tilgjengelig for interne apper (syfoservice).

Reply:
Tags er bare en måte å gruppere de på egentlig

Reply:
Fjerna egentlig bare hele den internal taggen